scripts/deploy_example_gcp

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPTS_DIR/common"
+
+source "$SCRIPTS_DIR/gcp_common"
+
+"$SCRIPTS_DIR/build_example" "hello_world"
+
+bazel_build_flags+=(
+    '--config=clang'
+)
+
+# Prerequisites:
+# - Google Cloud SDK
+# - Environment variable GCP_PROJECT_ID: id of the GCP project to use (eg oak-vm-testing).
+# - Environment variable GCP_ACCOUNT_FILE: path to the exported key for the service
+#   account that will be used to connect to GCP.
+# - Service account must be in the following IAM roles:
+#   - Compute Instance Admin (v1)
+#   - Compute Security Admin
+#   - Service Account User 
+
+# Create a minimal Ubuntu linux image on the GCP project.
+# The image will be named oak-gcp-hello-world.
+bazel run "${bazel_build_flags[@]}" "//examples/hello_world/gcp:deploy-${IMAGE_NAME}"
+
+# Authenticate as the service account
+gcloud auth activate-service-account \
+  --project="${GCP_PROJECT_ID}" \
+  --key-file="${GCP_ACCOUNT_FILE}"
+
+# Create Oak hello world VM instance and starts the Oak app on it.
+gcloud beta compute instances create "${INSTANCE_NAME}" \
+  --zone="${ZONE}" \
```

Comment:
Moved to one flag per line for clarity. I don't think a flags file will improve clarity furhter.